Documentation/block.md

```diff
@@ -99,3 +101,48 @@ kubectl delete -f mysql.yaml
 kubectl delete -n rook-ceph pool replicapool
 kubectl delete storageclass rook-ceph-block
 ```
+
+## Advanced Example: Erasure Coded Block Storage
+
+The below definitions are creating one replicated pool which is used for the RBD metadata, one erasure coded which is used for the data, and the actual StorageClass which is used for provisioning the storage.
+
+```yaml
+apiVersion: ceph.rook.io/v1beta1
+kind: Pool
+metadata:
+  name: replicaecpool
+  namespace: rook-ceph
+spec:
+  replicated:
+    size: 1
```

Comment:
To follow other documentation, we want the samples to show how it might be used in production (replicated size 3), and in the sample yaml we specify the size 1 to work in a minimal environment.